genie-agent/src/main/java/com/netflix/genie/agent/cli/JobRequestArgumentsImpl.java

```diff
@@ -0,0 +1,159 @@
+        converter = ArgumentConverters.FileConverter.class
+    )
+    private File jobDirectoryLocation = new File(DEFAULT_JOBS_DIRECTORY);
+
+    @Parameter(
+        names = {"--interactive"},
+        description = "Proxi standard {input,output,error} to the parent terminal, also disable console logging"
+    )
+    private boolean interactive;
+
+    @Parameter(
+        names = {"--archivalDisabled"},
+        description = "Disable server-side archival of logs (does not disable logs upload and short-term retrieval)"
+    )
+    private boolean archivalDisabled;
+
+    @Parameter(
+        names = {"--timeout"},
+        description = "Time (in seconds) after which a running job is forcefully terminated"
+    )
+    private Integer timeout;
+
+    @Parameter(
+        names = {"--jobId"},
+        description = "Unique job identifier"
+    )
+    private String jobId;
+
+    @Parameter(
+        names = {"--clusterCriterion"},
+        description = "Criterion for cluster selection, can be repeated (see CRITERION SYNTAX)",
+        converter = ArgumentConverters.CriterionConverter.class
+    )
+    private List<Criterion> clusterCriteria = Lists.newArrayList();
+
+    @Parameter(
+        names = {"--commandCriterion"},
+        description = "Criterion for command selection (see CRITERION SYNTAX)",
+        converter = ArgumentConverters.CriterionConverter.class
+    )
+    private Criterion commandCriterion;
+
+    @Parameter(
+        names = {"--applicationIds"},
+        description = "Override the applications a command normally depends on, can be repeated"
+    )
+    private List<String> applicationIds = Lists.newArrayList();
+
+    @Parameter(
+        names = {"--jobName"},
+        description = "Name of the job"
+    )
+    private String jobName;
+
+    @Parameter(
+        names = {"--user"},
+        description = "Username launching this job",
+        hidden = true // Not advertised to avoid abuse, but available for legitimate use cases
+    )
+    private String user = System.getProperty("user.name", "unknown-user");
+
+    @Parameter(
+        names = {"--email"},
+        description = "Email address where to send a job completion notification"
+    )
+    private String email;
+
+    @Parameter(
+        names = {"--grouping"},
+        description = "Group of jobs this job belongs to"
+    )
+    private String grouping;
+
+    @Parameter(
+        names = {"--groupingInstance"},
+        description = "Group instance this job belongs to"
+    )
+    private String groupingInstance;
+
+    @Parameter(
+        names = {"--jobDescription"},
+        description = "Job description"
+    )
+    private String jobDescription;
+
+    @Parameter(
+        names = {"--jobTag"},
+        description = "Job tag, can be repeated"
+    )
+    private Set<String> jobTags = Sets.newHashSet();
+
+    @Parameter(
+        names = {"--jobVersion"},
```

Comment:
Pretty much the stuff that ends up in the JobMetadata DTO is prefixed with `job`.
I cannot remove it from everything because i believe there is at least one field that conflicts.
So i instead added it to every field to make it explicit this is job tags, job description, job name, etc